examples/navier_stokes/ex7/Makefile

```diff
@@ -0,0 +1,33 @@
+######################################################################
+## Here specify the location of the IBAMR source and the location
+## where IBAMR has been built.
+IBAMR_SRC_DIR   = /Users/colegruninger/sfw/IBAMR_body_force/IBAMR
+IBAMR_BUILD_DIR = /Users/colegruninger/sfw/IBAMR_body_force/ibamr_bdy_force_opt
+
+######################################################################
+## Include variables specific to the particular IBAMR build.
+include $(IBAMR_BUILD_DIR)/config/make.inc
+
+## Needed for Xcode to capture compiler errors and warnings.
+ifdef XCODE_VERSION_ACTUAL
+CXXFLAGS += -fno-color-diagnostics
+endif
+
+######################################################################
+## Build the application.
+##
+## NOTE: The following assumes that all .cpp files in the present
+##       directory are used to build the executable.
+SRC = $(wildcard *.cpp)
+CPPFLAGS += -MD -MP
+PDIM = 3
+OBJS = $(SRC:%.cpp=%.o) $(IBAMR_LIB_3D) $(IBTK_LIB_3D)
+main3d: $(OBJS)
+	$(CXX) $(CXXFLAGS) $(LDFLAGS) $(OBJS) $(LDFLAGS) $(LIBS) -DNDIM=$(PDIM) -o main3d
+
+clean:
+	$(RM) main3d
+	$(RM) *.o *.lo *.objs *.ii *.int.c
+	$(RM) -r .libs
+
+-include $(SRC:%.cpp=%.d)
```

Comment:
The examples are built as part of IBAMR itself, not as independent programs. The build system as a whole needs to be aware of what's going on. Unfortunately this is not trivial to do unless you have prior experience doing it.

If you're willing, I'd still like you to work out how to do this instead of relying on one of the principal developers to write the correct code to connect this to the rest of IBAMR. Could you please look at how each `Makefile.am` is set up in 

https://github.com/IBAMR/IBAMR/commit/c0123bf893e91efe03494f3ed349932df87ff16e

and make the corresponding changes here? Once we add the automake files (the files ending in Makefile.am) and update the one in `navier_stokes` we should instruct the build system to use them by adding the correct lines to `configure.ac`. Once those files are in we can use the standard autotools rigamarole to add the correct `Makefile.in`s and appropriate changes to `configure` (those files are always automatically generated for us). I'll show you how to do that once we set up the non-generated files first.

Once that is done we get to add this example with CMake, which is a bit easier.

Reply:
Also, should this be a `navier_stokes` example or an `IB` example?

Reply:
Hmmmm... I think it might make a bit more sense for this to be an `IB` example. I'll implement that change as well.